packages/delegation-process/test/getDelegationContract.test.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable functional/no-let */
+import {
+  getMockAuthData,
+  getMockDelegation,
+} from "pagopa-interop-commons-test/index.js";
+import {
+  DelegationContractDocument,
+  DelegationContractId,
+  DelegationId,
+  delegationKind,
+  generateId,
+  operationForbidden,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  delegationContractNotFound,
+  delegationNotFound,
+} from "../src/model/domain/errors.js";
+import { addOneDelegation, delegationService } from "./utils.js";
+
+describe("getDelegationContract", () => {
+  const mockContract: DelegationContractDocument = {
```

Comment:
I suggest to use `actualDelegationContract` instead mockContract, in according with others test; in this way  expect assertion sound like more clear 
expect(returnedContract).toEqual(actualDelegationContract);